dev/loadgen/pkg/loadgen/executor.go

```diff
@@ -222,9 +222,12 @@ func (w *WsmanExecutor) StopAll(ctx context.Context) error {
 		if err != nil {
 			log.Warnf("could not get workspaces: %v", err)
 		} else {
-			if len(resp.GetStatus()) == 0 {
+			n := len(resp.GetStatus())
+			if n == 0 {
 				break
 			}
+			ex := resp.GetStatus()[0]
+			log.Infof("%d workspaces remaining, e.g. %s", n, ex.Id)
```

Comment:
some extra logging while stopping workspaces after a load test to view progress. Also include a workspace ID of a stopping workspace, to make it easy to inspect a workspace stuck in stopping